ride-share.rb

```diff
@@ -0,0 +1,175 @@
+driver_info = [
+      { driver_id: "DR0001",
```

Comment:
I'd suggest putting the 1st key-value pair on the a line separate from the opening {

```suggestion
      { 
        driver_id: "DR0001",
```